src/components/MyKiva/BorrowerStatusCard.vue

```diff
@@ -158,31 +117,24 @@ import {
 } from '#src/api/fixtures/LoanStatusEnum';
 
 const COMMENT_ID = 'comment';
-const DETAILS_ID = 'details';
 const SHARE_ID = 'share';
 
-const router = useRouter();
 const $kvTrackEvent = inject('$kvTrackEvent');
 
-const emit = defineEmits(['toggle-what-is-next', 'open-comment-modal', 'open-share-modal']);
+const emit = defineEmits(['open-comment-modal', 'open-share-modal', 'open-side-sheet']);
```

Comment:
Adding a comment is navigating the user to the BP, can we instead open the BP side sheet for mykiva and portfolio carousels? Let me know if this should be a follow-up ticket.

Reply:
I was referring to the app redirecting me to the BP after a comment was successfully added. Could we change the behavior to instead open the side sheet after a comment is successfully added from the borrower carousel?